frontend/src/components/PlanComs.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import Blog from "./Blog";
+
+function PlanComs() {
+  const [planComs, setPlanComs] = useState([]);
+  useEffect(() => {
+    fetch(
+      `${import.meta.env.VITE_BACKEND_URL ?? "http://localhost:5000"}/articles`
+    )
+      .then((response) => response.json())
+      .then((response) => setPlanComs(response));
+  }, []);
+  return (
+    <div>
+      {planComs.map((planCom, index) => {
+        return (
+          <Blog title={planCom.title} content={planCom.content} index={index} />
```

Comment:
key should be defined here. map index should not be used as a key : use an id instead